inlang/source-code/paraglide/paraglide-js/src/services/telemetry/events.ts

```diff
@@ -1,12 +1,22 @@
+import type { StackInfo } from "./stack-detection.js"
+
 /**
  * Typesafe telemetry events.
  *
  * Exists to avoid typos/always set the correct event name and properties.
  */
-export type TelemetryEvents =
-	| "PARAGLIDE-JS compile executed"
-	| "PARAGLIDE-JS init started"
-	| "PARAGLIDE-JS init project initialized"
-	| "PARAGLIDE-JS init added to devDependencies"
-	| "PARAGLIDE-JS init added compile commands"
-	| "PARAGLIDE-JS init finished"
+export type EventMap =
+	| {
+			event: "PARAGLIDE-JS compile executed"
+			properties: {
+				stack: StackInfo
+			}
+	  }
+	| {
+			event:
```

Comment:
Putting props in the type here seems unnecessarly complex. 

- the value of typesafety goes mostly for the event name to avoid telemetry problems
- properties regulaly change for events. that's ok

I would revert this change and keep things simple.